Formula/meta-package-manager.rb

```diff
@@ -0,0 +1,417 @@
+class MetaPackageManager < Formula
+  include Language::Python::Virtualenv
+
+  desc "🎁 wraps all package managers with a unifying CLI"
+  homepage "https://github.com/kdeldycke/meta-package-manager"
+  url "https://files.pythonhosted.org/packages/f5/28/5235fca899ead8059fbb35dd4bd786c1dbd4e1d0d65ae2d136540681a60c/meta_package_manager-5.13.0.tar.gz"
+  sha256 "8452718bbcec58dac5fd27691b1be621fcc0d25eb54cb3d3a7db89cfd7f2b1d3"
+  license "GPL-2.0-only"
+
+  depends_on "python@3.11"
+
+  resource "alabaster" do
+    url "https://files.pythonhosted.org/packages/94/71/a8ee96d1fd95ca04a0d2e2d9c4081dac4c2d2b12f7ddb899c8cb9bfd1532/alabaster-0.7.13.tar.gz"
+    sha256 "a27a4a084d5e690e16e01e03ad2b2e552c61a65469419b907243193de1a84ae2"
+  end
+
+  resource "attrs" do
+    url "https://files.pythonhosted.org/packages/21/31/3f468da74c7de4fcf9b25591e682856389b3400b4b62f201e65f15ea3e07/attrs-22.2.0.tar.gz"
+    sha256 "c9227bfc2f01993c03f68db37d1d15c9690188323c067c641f1a35ca58185f99"
+  end
+
+  resource "babel" do
```

Comment:
```suggestion
  resource "Babel" do
```

Reply:
Also, several PyPI resources can be converted to formula dependencies. For example, `docutils` and `pyyaml`. Can I request you to identify all, make them formula dependencies, and add an entry to [`pypi_formula_mappings.json`](https://github.com/Homebrew/homebrew-core/blob/master/pypi_formula_mappings.json)? The PyPI package names should be listed in `exclude_packages`.

Reply:
Oh OK. I see; the policy here is to rely as much as possible on formula already managed by `brew`. I'll make the necessary changes then.

Reply:
Just fixed the case of Python dependencies, and added direct dependencies on other Formula, namely: `docutils`, `pygments`, `python-tabulate`, `python-typing-extensions` and `pyyaml`.

`pypi_formula_mappings.json` has also been updated to reflect these changes.